stf.build/include/top.xml

```diff
@@ -114,7 +114,7 @@ limitations under the License.
 		Targets to check for and download prereqs.
 		All prereqs are installed under the same ${prereqs_root} directory.
 	-->
-	<target name="configure" depends="check-for-wget,
+	<target name="configure" depends="check-for-download-tool,
```

Comment:
Is there check-for-download-tool target already?

Reply:
Yes, `check-for-download-tool` target is part of this pr.

Reply:
yeah, seems I didn't finish the whole PR.